tests/regression/04-mutex/03-munge_rc.c

```diff
@@ -12,15 +12,15 @@ void munge(pthread_mutex_t *m) {
 }
 
 void *t_fun(void *arg) {
-  munge(&mutex2); // NOWARN!
+  munge(&mutex2); // NOWARN
```

Comment:
Why `NOWARN` here instead of `NORACE`?

Reply:
There are a couple of places like this.

Reply:
I left them like that because those are slightly unrelated to the question of races and comparing the `rc`/`nr` versions it's not where a race is happening anyway. I didn't want to remove them either because supposedly there's something we want to check, but that's more of a user experience question. Like, why do we insist on checking that Goblint doesn't tell the user where the racing function is called from.

Reply:
No, there is nothing we want to check there. The 04-mutex category is probably the oldest regression test category and my understanding then was to very narrowly follow goblint's current behavior. It would be much better if the tests make sense independently of what goblint does (except TODO annotations).